packages/eslint-plugin/rules/href_or_on_click.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  meta: {
+    fixable: null,
+  },
+  create(context) {
+    return {
+      JSXOpeningElement(node) {
+        if (
+          node.name.type !== 'JSXIdentifier' ||
+          node.name.name !== 'EuiButton'
```

Comment:
So if we want to use this rule for all our button/link style components, do we just append to this list like:
```suggestion
          node.name.name !== 'EuiButton' ||
          node.name.name !== 'EuiButtonEmpty' ||
          node.name.name !== 'EuiLink'
```

Reply:
That's not how I'd do it, but essentially, yes

Reply:
What would be the more efficient way? Just trying to figure out how to continue to add to this.

Reply:
I've updated the code to use an array of component names that is checked against.

Reply:
https://github.com/elastic/eui/pull/2218/files#diff-ddf6696d70d8327aa2404debe43a9acfR1